caffe2/operators/quantized/int8_simd.h

```diff
@@ -19,10 +19,11 @@
 
 #if defined(__SSE4_2__) && defined(__clang__)
 #define INT8_NEON_SIMD
+#include "NEON_2_SSE.h"
+#endif
 
-#include "neon2sse.h"
+#if !defined(GEMMLOWP_SSE4) && !defined(GEMMLOWP_NEON)
```

Comment:
The following functions(e.g. Dup() and BitAnd()) are already in GEMMLOWP_NEON and GEMMLOWP_SSE4 header.